tianshou/utils/logger/base.py

```diff
@@ -172,6 +176,9 @@ def prepare_dict_for_logging(
     def write(self, step_type: str, step: int, data: dict[str, VALID_LOG_VALS_TYPE]) -> None:
         """The LazyLogger writes nothing."""
 
+    def finalize(self) -> None:
+        pass
+
```

Comment:
Do we need the extra implementation? Now mypy is complaining about a duplicate. I first tried to have an empty non abstract method in the base class, but then ruff was complaining..

Reply:
I removed the ruff rule that caused the complaining and now also removed the abstractmethod

Reply:
Ah, well, since it now actually is implemented in all loggers, the non-abstract implementation should be removed from BaseLogger. I'll do that now